eventkit_cloud/tasks/tests/test_scheduled_tasks.py

```diff
@@ -77,6 +81,26 @@ def test_expire_runs(self, send_email):
             self.assertEqual(3, ExportRun.objects.all().count())
 
 
+class TestCheckProviderAvailabilityTask(TestCase):
+
+    @patch('eventkit_cloud.utils.provider_check.perform_provider_check')
+    def test_check_provider_availability(self, perform_provider_check_mock):
+        perform_provider_check_mock.return_value = json.dumps(CheckResults.SUCCESS.value[0])
+        first_provider = DataProvider.objects.create(slug='first_provider', name='first_provider')
+        second_provider = DataProvider.objects.create(slug='second_provider', name='second_provider')
+        DataProviderStatus.objects.create(related_provider=first_provider)
+
+        check_provider_availability()
+
+        perform_provider_check_mock.assert_has_calls([call(first_provider, None), call(second_provider, None)])
+        statuses = DataProviderStatus.objects.filter(related_provider=first_provider)
+        self.assertEqual(len(statuses), 2)
+        most_recent_first_provider_status = statuses.order_by('-id')[0]
+        self.assertEqual(most_recent_first_provider_status.status, 'SUCCESS')
+        self.assertEqual(most_recent_first_provider_status.status_type, 'SUCCESS')
+        self.assertEqual(most_recent_first_provider_status.message, 'Export should proceed without issues.')
```

Comment:
You might want to change these to:
```
self.assertEqual(most_recent_first_provider_status.status, CheckResults.SUCCESS['status'])
self.assertEqual(most_recent_first_provider_status.status_type, CheckResults.SUCCESS['type'])
self.assertEqual(most_recent_first_provider_status.message, CheckResults.SUCCESS['message'])
```
Works better with localization, and also works if we ever want to change the message value.